package.json

```diff
@@ -46,6 +46,7 @@
   "unpkg": "dist/antd.min.js",
   "typings": "lib/index.d.ts",
   "scripts": {
+    "prepare": "husky install",
```

Comment:
这样写 没有 husky 不会挂吗？

Reply:
`husky`升级文档是这样指导的。并且：

```
prepare (since npm@4.0.0)

Runs any time before the package is packed, i.e. during npm publish and npm pack

Runs BEFORE the package is packed

Runs BEFORE the package is published

Runs on local npm install without any arguments

Run AFTER prepublish, but BEFORE prepublishOnly

NOTE: If a package being installed through git contains a prepare script, its dependencies and devDependencies will be installed, and the prepare script will be run, before the package is packaged and installed.

As of npm@7 these scripts run in the background
```